client/src/components/Overview/Overview.jsx

```diff
@@ -1,14 +1,64 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
 import ImageGallery from 'Overview/ImageGallery.jsx';
 import ProductInformation from 'Overview/ProductInformation.jsx';
+import testData from 'tests/testData.js';
+
+function Overview({ productId }) {
+  const [product, setProduct] = useState({});
+  const [styles, setStyles] = useState([]);
+  const [selectedStyleId, setSelectedStyleId] = useState();
+
+  useEffect(() => {
+    // TODO: account for unmounting
+    async function fetchData() {
+      try {
+        const [productResponse, stylesResponse] = await Promise.all([
+          axios.get(`/products/${productId}`),
+          axios.get(`/products/${productId}/styles`),
+        ]);
+
+        const sortedStyles = stylesResponse.data.results.sort((style1, style2) => (
+          style1.styled_id - style2.styled_id
+        ));
+
+        setProduct(productResponse.data);
+        setStyles(sortedStyles);
+        setSelectedStyleId(sortedStyles.find(style => style['default?']).style_id);
```

Comment:
I believe for the style guide it wants you to have parens even if its only 1 arg

Reply:
Yeah, I have noticed that with the style guide. I feel like it makes things more cluttered and less legible but I'm fine with adhering to that style. I'll push a change

Reply:
I think it makes it easier to see its a func at a glance but I do agree that its more cluttered

Reply:
yeah thinking about it, I agree with you. Helps eliminate the symbol soup that can occur.